### DISCUSSION TITLE
preview specific format (when several are specified)

### DISCUSSION THREAD

**Discussion by skwde** [quarto-dev/quarto-cli#10312] [Q&A] [answered]
### Description

I have following `_quarto.yml`

```yml
format:
  latex: default
  pdf:
    documentclass: scrartcl
```

and the `text.qmd`

````qmd
# Test

abc
````

When I run `quarto preview test.qmd --no-browser` the preview is available on http://localhost:5293/test.tex.

Some observations:

- Using `--render pdf` seems to have no effect.
- Changing the order of formats renders to pdf and also shows a different preview url http://localhost:7071/web/viewer.html

How do I tell `quarto preview` what preview I actually want to have without relying on the `format` order?

I use Quarto version 1.5.54 in a devcontainer running debian bookworm.

**Comment by Steinthal** [comment, marked as answer]
Use `--to <format>`, e.g. `quarto preview test.qmd --to pdf`

See ["All Formats"](https://quarto.org/docs/output-formats/all-formats.html) in the Quarto documentation.
